apps/server/src/adapters/mx.ts

```diff
@@ -173,24 +172,29 @@ export class MxAdapter implements WidgetAdapter {
     userId: string
   ): Promise<Connection> {
     let ret
-    if (request.job_type === 'verification') {
-      ret = await this.apiClient.verifyMember(request.id, userId)
-    } else if (request.job_type === 'aggregate_identity') {
-      ret = await this.apiClient.identifyMember(request.id, userId, {
-        data: { member: { include_transactions: true } }
-      })
-    } else if (request.job_type === 'aggregate_extendedhistory') {
-      ret = await this.apiClient.extendHistory(request.id, userId)
-    } else {
-      ret = await this.apiClient.aggregateMember(request.id, userId)
-    }
-
-    if (ret?.data?.error?.message === EXTENDED_HISTORY_NOT_SUPPORTED_MSG) {
-      ret = await this.apiClient.aggregateMember(request.id, userId)
-    }
 
-    if (ret.data?.error) {
-      return { id: request.id, error_message: ret.data.error.message }
+    try {
```

Comment:
The `mx-platform-node` library has updated changes, including how it returns errors, if/when they happen. 

Old code returned a 200, but included an `error` object in the body. 

The new library throws an AxiosError, thus the new try/catch pattern.